v3/generate.go

```diff
@@ -36,23 +37,29 @@ func Generate() {
 	}
 
 	for _, versionStr := range versionStrs {
-		clientBytes, err := generateClientYml(swagger, versionStr)
+		swaggerCopy := duplicateMap(swagger)
+
+		if versionStr == "v3.0" {
+			deleteV31Definitions(swaggerCopy)
```

Comment:
I added 2 definitions to `full-v3.yml` - `Disability` and `PreferredName` - that are not required in API v3.0 schema. I've added logic here for each version to use a unique swagger copy to delete any definitions if required before YAML generations.